torch/testing/_asserts.py

```diff
@@ -243,11 +227,8 @@ def _check_values_close(
     if not torch.any(mismatches):
         return None
 
-    trace = _trace_mismatches(a, b, mismatches)
     return AssertionError(
-        f"With rtol={rtol} and atol={atol}, "
-        f"found {trace.abs} different element(s) out of {trace.total} ({trace.rel:.1%}). "
-        f"The greatest difference of {trace.diff} ({trace.a} vs. {trace.b}) occurred at index {trace.idx}"
+        f"Tensors are not close according to rtol={rtol} and ator={atol}:\n\n{_trace_mismatches(a, b, mismatches)}"
```

Comment:
"ator" -> "atol"

Reply:
"Tensors are not close:"

or

"Tensors are not close!" 

(see above)

Reply:
You're correct that if we follow the current isclose() logic we don't need to take this recommendation, because the atol and rtol needed are just the greatest absolute difference and greatest relative difference. Maybe instead we can say:

"Greatest absolute difference: {x} (up to {y} allowed)"

or:

"
Greatest absolute difference: {x}
Allowed absolute difference: {y}
"